### DISCUSSION TITLE
LUA Recreations of Removed JSE Features

### DISCUSSION THREAD

**Discussion by JordanSantiagoYT** [JordanSantiagoYT/FNF-JS-Engine#429] [General]
hello. this is a page where you can find lua scripts that accurately recreate any removed feature from JSE if you want it.

[More Max Health and 0% Health Limit on Practice](https://github.com/user-attachments/files/16134323/More.Max.Health.and.Zero.Health.Limit.on.Practice.zip)
[Double Note Ghosts (made by Laztrix)](https://github.com/user-attachments/files/16134324/Double.Note.Ghosts.zip)
Hide ScoreTxt: `setProperty('scoreTxt.visible', false)`
Score Text Size: `setTextSize('scoreTxt', 18)`
[Time Text Bounce](https://drive.google.com/file/d/12js4S-9fnZLz05TIV1Y1WK41-17pcb7z/view?usp=drive_link)
[Playback Rate Controls](https://drive.google.com/file/d/1quX8hk5W--azXraMl7g0FWuVOIkLPVCL/view?usp=drive_link)
[Camera Note Movement (made by An Ammar)](https://drive.google.com/file/d/1bgIM9SP-Gmc-06t7qy_R4lEX54V88sVC/view?usp=drive_link)
[Only Sicks](https://drive.google.com/file/d/1LtnqnZTDco85w5WPwn3LCMznkbyAuxQ5/view?usp=drive_link)
[YouTube Watermark](https://drive.google.com/file/d/1QnYpYQXB3d7wuWUf6hKf-Bvi54oPy2d7/view?usp=sharing)
[Rating Counter](https://drive.google.com/file/d/18p_JG66-_TwEJDsBCkwhUSJr3mIMNUHU/view?usp=sharing)
[Render Info (recreated by Just Another Random)](https://drive.google.com/file/d/1OzBXa6MkOr6bTm4wG7Utz4sAaFn1cBUo/view?usp=sharing)
[Random Botplay Text (includes the fnaf jumpscare)](https://drive.google.com/drive/folders/1GI43ySyfpbs2YFmWjz-cZ124T6LZyzOb?usp=drive_link)
[Health Bar Intro Tween](https://drive.google.com/file/d/1h2sJ2ob3TJnkOQR3td_iCazwo_VOC6m1/view?usp=sharing)
[Showcase Mode](https://drive.google.com/file/d/15c39IeSsdHrOD2A17xyRd0vZpGbFg717/view)

NOTE: Rating Counter, Double Note Ghosts and Showcase Mode are not supported on most engines!

**Comment by FinderFromMac** [comment]
yay

**Comment by FuseIsHere813** [comment]
For hiding score text, use this code in your LUA script:

`setProperty('scoreTxt.visible', false)`

**Comment by CoolDudeT** [comment]
Can someone modify the double note ghost script to not make the camera zoom in when there's double notes?

**Reply by TheStinkern** [reply]
it's an option in the gameplay options

**Reply by JordanSantiagoYT** [reply]
> it's an option in the gameplay options

erm..

**Reply by CoolDudeT** [reply]
> it's an option in the gameplay options

yeah no

**Comment by TheStinkern** [comment]
to make every object fade out thats in a camera do this:

for camGame: doTweenAlpha('Fade', 'camGame', 0, 'expoOut')

for camHud: doTweenAlpha('Fade', 'camHud', 0, 'expoOut')

for camOther: doTweenAlpha('Fade', 'camOther', 0, 'expoOut')

**Reply by TheStinkern** [reply]
i just noticed that It says "removed js engine features"

I'm smart at reading

**Comment by nate902011** [comment]
wow you remember this page existed.

**Comment by JordanSantiagoYT** [comment]
The Showcase Mode script has been updated, fixing an issue where it wouldn't work with H-Slice due to that engine not having Note Multipliers.
